thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/AnomalyDetectionUtils.java

```diff
@@ -12,4 +19,36 @@
   public static long getBucketInMillis(int bucketSize, TimeUnit bucketUnit) {
     return bucketUnit.toMillis(bucketSize);
   }
+
+  private static Splitter SEMICOLON_SPLITTER = Splitter.on(";").omitEmptyStrings();
+  private static Splitter EQUALS_SPLITTER = Splitter.on("=").omitEmptyStrings();
+  private static Joiner SEMICOLON = Joiner.on(";");
+  private static Joiner EQUALS = Joiner.on("=");
+
+  // the following two functions encode and decode properties are moved from class
+  // com.linkedin.thirdeye.controller.mp.function.ThirdEyeAnomalyFunctionUtil
```

Comment:
You don't need to JavaDoc where is this method moved from; however, please JavaDoc what does this method do.